django/cantusdb_project/main_app/views/chant.py

```diff
@@ -95,12 +215,40 @@ def get_context_data(self, **kwargs):
             context["syllabized_text_with_melody"] = text_and_mel
 
         # If chant has a cantus ID, Create table of concordances
+        context["concordances_loaded_successfully"] = True
         if chant.cantus_id:
```

Comment:
This `if` statement is still really long, so it's hard to see what it's doing. Any chance you could break this out into a separate function?